lib/logger/logger.go

```diff
@@ -339,6 +340,25 @@ func (r *recorder) Since(t time.Time) []Line {
 	return nil
 }
 
+func (r *recorder) All() []Line {
```

Comment:
I think this method is not really necessary as it's equivalent to `Since(time.Time{})` (i.e., everything since the zero time).

(Removing it here and from the logger interface will also make the build pass, as it currently fails due to the test version of the recorder not fulfilling the updated interface.)